Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -3142,40 +3142,42 @@ def panorama_get_pcap_command():
 
 def prettify_applications_arr(applications_arr):
     pretty_application_arr = []
+    if not isinstance(applications_arr, list):
+        applications_arr = [applications_arr]
     for i in range(len(applications_arr)):
         application = applications_arr[i]
         pretty_application_arr.append({
-            'SubCategory': application['subcategory'],
-            'Risk': application['risk'],
-            'Technology': application['technology'],
-            'Name': application['@name'],
-            'Description': application['description'],
-            'Id': application['@id']
+            'SubCategory': application.get('subcategory'),
+            'Risk': application.get('risk'),
+            'Technology': application.get('technology'),
+            'Name': application.get('@name'),
+            'Description': application.get('description'),
         })
     return pretty_application_arr
 
 
 @logger
 def panorama_list_applications():
     params = {
-        'type': 'op',
-        'command': '<show><objects></objects></show>',
+        'type': 'config',
+        'action': 'get',
+        'xpath': XPATH_OBJECTS + "application/entry",
         'key': API_KEY
     }
+    demisto.log(params['xpath'])
```

Comment:
let's remove this

Reply:
I'm re reviewing this issue, will update today this as well and request review again.